src/workflows/annotation/scgpt_integration_knn/integration_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# get the root of the directory
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# ensure that the command below is run from the root of the repository
+cd "$REPO_ROOT"
+
+export NXF_VER=21.10.6
+
+viash ns build -q scgpt_integration_knn
```

Comment:
We used to do this, but recently it was clear that this is not the best idea. Lets remove it.
```suggestion
```